package.json

```diff
@@ -65,7 +65,6 @@
     "lru_map": "^0.3.3",
     "marked": "^4.0.17",
     "multiple-select": "^1.5.2",
-    "nw-vue-devtools-prebuilt": "^0.0.10",
```

Comment:
Why is this removed?

Reply:
I cannot install the software nw-vue-devtools-prebuilt. The function of this software is to debug applications. I will try again. If it still cannot be installed, I will give up on it because I want to see the result of the application. I will directly generate a debugging version of osx64, which only takes 15 seconds.